src/pages/projects.js

```diff
@@ -422,6 +422,41 @@ const ProjectDescriptions = () => (
 
     <TitleWithAnchor headerClassName="projects-title projects-common-title">Emerging Projects</TitleWithAnchor>
 
+    <div className="project-box">
```

Comment:
For now I'd say it's better to append to the list of emerging projects rather than prepend.

Reply:
Sorry, seems like the list is currently in alphabetical order. wfm.